### DISCUSSION TITLE
Language supported ?

### DISCUSSION THREAD

**Discussion by ainthek** [shivasurya/code-pathfinder#168] [General]
Please what languages are supoorted ? it is not written anywhere, seems like only java ? any plants to supoort other languages ?

**Comment by shivasurya** [comment]
Are there any other programming languages you'd like us to support? Let me know!

Currently, the Java language is supported, and Kotlin support is coming soon! Since Java and Kotlin have similar semantics (though Kotlin has a more concise syntax), adding Kotlin is a natural next step. I'll update the documentation and contributions are always welcome!

**Comment by ainthek** [comment]
csharp, I do not love it just need it ;-(
